### PR TITLE
ref(package.json): Move some deps into devDeps

### PR DESCRIPTION
**Pull request by jeffkwoh**

### CODE REVIEW

**Comment by EvanPurkhiser** [comment]
I don't think we can do this since babel is used at build time.

**Comment by dashed** [comment]
I think there's a specific reason why these deps aren't in `devDependencies`. It has something to do with how they're packaged within `getsentry/getsentry`.

File: package.json [hunk 1]
```diff
@@ -7,46 +7,15 @@
     "url": "git://github.com/getsentry/sentry.git"
   },
   "dependencies": {
-    "@babel/core": "~7.7.2",
```
Comment (billyvg) [inline package.json]: IIRC these are required dependencies for CI/docker to build our assets

**Comment by jeffkwoh** [comment]
Closing PR as refactoring the deps no longer seems feasible as they are used in build time. @reviewers, thanks for shedding light on this.

**Pull request closed without merging**